templates/projects/list.tmpl

```diff
@@ -1,4 +1,4 @@
-{{if .CanWriteProjects}}
+{{if and $.CanWriteProjects (not $.Repository.IsArchived)}}
```

Comment:
Isn't that a regression for org/user projects?
Do they have a `Repository` attribute?

Reply:
Okay, I just tested it.
For whatever reason, it works.

Reply:
wait. $.Repository maybe nil?

Reply:
No, for some reason it isn't.
As I said, I tested it, it works.
I don't know either why.

Reply:
But it's really wrong? Why not fix it?

Reply:
It's not wrong.

https://pkg.go.dev/text/template#Template.Option

Reply:
That's the weird thing:
I think it's invalid too, but for some weird reason it works.
So, let's leave the clean up of that for another PR?